### DISCUSSION TITLE
Hey is the auto fix feature disabled?

### DISCUSSION THREAD

**Discussion by saicharank** [pmd/pmd#4504] [Q&A]
wanted to check if auto fix feature is removed from latest versions.

**Comment by jsotuyod** [comment]
The feature was never fully implemented. We worked on some PoCs and figuring out how it would be integrated into other tools, but never got it to production grade. This is still in our backlog 'though.

**Comment by saicharank** [comment]
Thanks. But do we have unused classes predefined rule?

**Reply by jsotuyod** [reply]
No, the code has been removed / deprecated, as PMD 7 heavily refactored several places (including but not limited to the Java grammar). We took the learnings, and I personally do expect this to be implemented eventually 'though, but help in achieving this is most definitely welcomed.
